src/mpsc/errors.rs

```diff
@@ -1,6 +1,26 @@
 //! Errors returned by channels.
 use core::fmt;
 
+/// Error returned by the [`Sender::send_timeout`] or [`Sender::send_ref_timeout`] 
```

Comment:
whoops, rustfmt is mad about trailing whitespace here (which i introduced, my bad):
```suggestion
/// Error returned by the [`Sender::send_timeout`] or [`Sender::send_ref_timeout`]
```